pkg/abstractions/map/map_redis.go

```diff
@@ -26,7 +32,15 @@ func NewRedisMapService(rdb *common.RedisClient) (MapService, error) {
 func (m *RedisMapService) MapSet(ctx context.Context, in *pb.MapSetRequest) (*pb.MapSetResponse, error) {
 	authInfo, _ := auth.AuthInfoFromContext(ctx)
 
-	err := m.rdb.Set(ctx, Keys.MapEntry(authInfo.Workspace.Name, in.Name, in.Key), in.Value, 0).Err()
+	if len(in.Value) > maxMapValueSize {
+		return &pb.MapSetResponse{Ok: false, ErrMsg: "Value cannot be larger than 1 MiB"}, nil
+	}
+
+	if time.Duration(in.Ttl)*time.Second > maxMapValueTtls {
+		return &pb.MapSetResponse{Ok: false, ErrMsg: "TTL cannot be larger than 1 week"}, nil
```

Comment:
Change to say longer than 1 week probably

Reply:
Updated.